unity/PitayaExample/Assets/Pitaya/PitayaBinding.cs

```diff
@@ -62,6 +62,8 @@ public static class PitayaBinding
         private static readonly Dictionary<IntPtr, int> EventHandlersIds = new Dictionary<IntPtr, int>();
         private static PitayaLogLevel _currentLogLevel = PitayaLogLevel.Disable;
         private static bool IsNativeLibInitialized;
+        
+        public static IPitayaQueueDispatcher QueueDispatcher { get; set; } = new NullPitayaQueueDispatcher();
```

Comment:
@felippeduran shouldn't this be a MainQueueDispatcher by default? I guess many games will start getting "UI should not be called from background thread" if suddenly some events start getting called from the background. Or is it that in this case the methods are already being called from the main thread anyway?

Reply:
Or, I just realized that this is just an example... not sure if it still applies though. :)